Resources/docs/sidebar_navigation.md

```diff
@@ -83,6 +83,10 @@ class MenuBuilderSubscriber implements EventSubscriberInterface
         foreach ($items as $item) {
             if ($item->hasChildren()) {
                 $this->activateByRoute($route, $item->getChildren());
+                if (null !== $item->getActiveChild())
```

Comment:
could you add a comment here, what this exactly does, so people know what this one is for?!
having two active items doesn't look correct on the first glance

Reply:
It is useful while using this method in breadcrumb event to activate whole path from active child and all parents. It will generate working breadcrumb. Without it, breadcrumb doesn't show on MenuItems nested deeper than first level.